docker-scripts/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:18.04
+
+#Build in this directory with sudo docker build . or docker build . (depending on your OS)
+ENV GOROOT=/usr/local/go
+ENV GOPATH=/opt/sensu
+ENV PATH=$GOPATH:/bin:$GOROOT/bin:/opt/sensu/bin/:$PATH
+
+WORKDIR /opt/sensu
+COPY ./sensu-entrypoint.sh /opt/sensu/bin/
+
+RUN apt update &&\
+    apt -y upgrade &&\
+    apt -y install git wget
+
+#Download and install Go 1.13.3 for Go modules.
+RUN wget https://dl.google.com/go/go1.13.3.linux-amd64.tar.gz && tar -xvf go1.13.3.linux-amd64.tar.gz &&\
```

Comment:
to make it easier to update it might be good to use an ENV var for the version number to reduce the number of places that are required when we want to build with a newer golang.

Reply:
I'll fix.

Reply:
See latest commit https://github.com/sensu/sensu-go/pull/3456/commits/0713c3dd0693466607e4c1dc2d82a56dc22dfcc5